srcpkgs/OpenRCT2/template

```diff
@@ -71,7 +72,7 @@ if [ "$XBPS_CHECK_PKGS" ]; then
 	configure_args+=" -DWITH_TESTS=1 -DSYSTEM_GTEST=1"
 	# download replays, used in tests
 	distfiles+=" https://github.com/OpenRCT2/replays/releases/download/v${_replays_version}/replays.zip"
-	checksum+=" bf2184cc6ace156402a26d3e4693b6363b888fe3038daa59e33b202ce0efe0d9"
+	checksum+=" 89156436e51413f180a12d4647406193ac44a509b1ad64d57705f0ae5125b093"
```

Comment:
let's also edit skip_extraction here instead of always skipping it, in the future it would be nice if pkglint aborted the build if something that doesn't exist is in skip_extraction